docs/reference/modules/piece-type.md

```diff
@@ -0,0 +1,272 @@
+---
+extends: '@apostrophecms/doc-type'
+---
+
+# `@apostrophecms/piece-type`
+
+This module is the foundation for all [piece types](/guide/pieces.md) in Apostrophe. It is not typically configured or referenced in project code directly since each piece type should be managed independently in most cases. For example, the options documented below would be configured on a custom piece type, e.g., `article`, rather this piece type base module.
+
+The only reason to configure this module directly would be to apply the changes to *every* piece type, including those Apostrophe core (e.g., `@apostrophecms/user`).
+
+**Extends:** `{{ $frontmatter.extends }}`
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+| [`autopublish`](#autopublish) | Boolean | Set to `true` to publish all saved edits immediately. |
+| [`label`](#label-for-doc-types) | String | The human-readable label for the doc type. |
+| [`localized`](#localized) | Boolean | Set to `false` to exclude the doc type in the locale system. |
+| [`perPage`](#perpage) | Integer | The number of pieces to include on `req.data.pieces` in each page. |
+| [`pluralLabel`](#plurallabel) | String | The plural readable label for the piece type. |
+| [`publicApiProjection`](#publicapiprojection) | Object | Piece fields to make available via a public REST API route. |
+| [`quickCreate`](#quickcreate) | Boolean | Set to `true` to add the piece type to the quick create menu. |
+| [`searchable`](#searchable) | Boolean | Set to `false` to remove the piece type from search results. |
+| `showCreate` | Boolean | Set to `false` to disable UI related to creating new pieces of that type. |
+| `showArchive` | Boolean | Set to `false` to disable UI related to archiving pieces of that type. |
+| `showDiscardDraft` | Boolean | Set to `false` to disable UI related to discarding draft pieces of that type. |
+| `showDismissSubmission` | Boolean | Set to `false` to disable UI related to dismissing draft submissions for pieces of that type. |
+| `singleton` | Boolean | Set to `true` to ensure that no one can create a new piece of that type. The global doc as only one should ever exist. |
+| [`sort`](#sort) | Object | The value for a piece type's default sort order query builder. |
+
+### `autopublish`
```

Comment:
Also cover `localized` and the distinction between the two, discussed in my earlier comment.